### ISSUE TITLE
not working for this command

### ISSUE THREAD

**Issue by parsibox** [incarnate/curl-to-php#11]
not work for this command
```
curl -v -X POST -F "name=4444" -F "service-key=zzzz" -F ufile=@"/data/publicweb/xxx.zip" "http://1.0.13.13
2:56/upload"
```

**Comment by incarnate** [comment]
Thanks for raising the issue, currently the script does support `-d` (application/x-www-form-urlencoded) but not `-F` (multipart/form-data ).

I'll have a look at this, but a PR is welcome if anyone is up for the challenge.

**Comment by incarnate** [comment]
Resolved by #27
